### ISSUE TITLE
Command line parameters

### ISSUE THREAD

**Issue by JanPietrzyk** [shopwareLabs/psh#36]
It would be nice to set or even overwrite placeholders through the command line.

```
> ./psh.phar init --env=dev
> ./psh.phar init --env="string that must be parsed"
> ./psh.phar init --env="echo my_script.sh" # should **not** execute the script depending on the type of variable
```

Would then result for

```
echo __ENV__ 
```
in 

```
> dev
```

Remember the issue with replacing dynamic variables with constant values. All CLI-Parameters should always be  created as constants

This idea was brought forward by @janwalther in #18. Therefore it is labeled question, as in "if this is not what you want, please comment" ;)

**Comment by janwalther** [comment]
This would be nice. In my example I would like to provision docker containers. The setup instrcutions are always the same but different containers shall be used. With command line arguments I could call
`./psh.phar provision --container=microservice1`
And in provision.sh there would be
`docker-compose run __CONTAINER__ composer update`

**Comment by JanPietrzyk** [comment]
fixed by #48
